src/frontend/context/SimuleringContext.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+
+import constate from 'constate';
+import dayjs from 'dayjs';
+
+import { useHttp } from '@navikt/familie-http';
+import { RessursStatus, Ressurs } from '@navikt/familie-typer';
+
+import { aktivVedtakPåBehandling } from '../api/fagsak';
+import { IBehandling } from '../typer/behandling';
+import { ISimuleringPeriode, ISimuleringDTO } from '../typer/simulering';
+import familieDayjs from '../utils/familieDayjs';
+
+interface IProps {
+    åpenBehandling: IBehandling;
+}
+
+const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProps) => {
+    const { request } = useHttp();
+    const aktivtVedtak = aktivVedtakPåBehandling(åpenBehandling);
+    const [simuleringResultat, settSimuleringResultat] = useState<Ressurs<ISimuleringDTO>>({
+        status: RessursStatus.HENTER,
+    });
+
+    useEffect(() => {
+        request<IBehandling, ISimuleringDTO>({
+            method: 'GET',
+            url: `/familie-ba-sak/api/simulering/${aktivtVedtak?.id}`,
+        }).then(response => {
+            settSimuleringResultat(response);
+        });
+    }, [aktivtVedtak]);
+
+    const hentPeriodelisteMedTommePerioder = (
+        perioder: ISimuleringPeriode[]
+    ): ISimuleringPeriode[] => {
+        const fomDatoer = perioder
+            .map(periode => periode.fom)
+            .sort((a, b) => (dayjs(a).isAfter(dayjs(b)) ? 1 : -1));
+        const førstePeriode = fomDatoer[0];
+        const sistePeriode = fomDatoer[-1];
```

Comment:
Fungerer dette for å hente siste element? 
Skulle det vært fomDatoer[fomDatoer.length -1] ?